src/agents/agent.js

```diff
@@ -25,7 +24,7 @@ class Agent {
 	}
 
 	getAssistantId() {
-		return WAPUU_ASSISTANT_ID;
```

Comment:
@glendaviesnz was there a reason for changing the assistant ID?

Reply:
I'm going to change this back for now. I think this breaks the demo that uses the Assistants API, but the CLI still seems to be working.